components/EditCoverSection.tsx

```diff
@@ -87,7 +87,48 @@ const EditCoverSection = () => {
     }
   };
 
+  const imageRef = React.useRef<any>(null);
+
+  const handleViewClick = () => {
+    if (imageRef?.current instanceof HTMLImageElement) {
+      imageRef.current.click();
+    }
+  };
+
   const handleView = (e: any) => {
+    const imageUrl = e.target.src;
+    console.log(e.target);
+    const lightbox = document.createElement("div");
+    lightbox.style.position = "fixed";
```

Comment:
Instead of adding multiple lines for style we can create a class or component in our global.css